AdLerDokumentation/Writerside/topics/EKJ0001.md

```diff
@@ -0,0 +1,17 @@
+# Pausenhinweis anzeigen
+
+
+# Definition:
+
+Nach Ablauf einer bestimmten Zeit muss die Engine einen Hinweis auf die zugehörige Pausenempfehlung anzeigen.
```

Comment:
Hier könnte man vll noch einen Zustand definieren sowas wie in Betrieb und dann wie folgt formulieren: 

Solange sich die Engine im Zustand Betrieb befindet muss die Engine nach den folgenden Zeitinterwallen einen Pausenhinweis anzeigen: 30 min, 120 min ..(und was ihr halt noch so braucht)

-> Dazu sollte aber der Zustand Betrieb definiert sein.

Reply:
Sprech ich im daily dann mal an